config/constants.yml

```diff
@@ -7,7 +7,7 @@
 ---
 
 # App
-TERMINUS_VERSION: '2.6.1-dev'
+TERMINUS_VERSION: '2.6.1'
```

Comment:
Let's merge this in as 2.6.1-dev first, and tag a release that does nothing but bump the version in a follow-on PR

Reply:
As mentioned in slack, we don't need a release for this yet.

Reply:
Returned -dev to the version number. So we merge this in and don't tag it as a release?